front/app/components/admin/PostManager/components/FilterSidebar/statuses/ScreeningStatusFilter.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+
+import { IconTooltip, Box, Tooltip } from '@citizenlab/cl2-component-library';
+import ColorIndicator from 'component-library/components/ColorIndicator';
+import { useDrop } from 'react-dnd';
+import { useParams } from 'react-router-dom';
+import styled from 'styled-components';
+
+import { IIdeaStatusData } from 'api/idea_statuses/types';
+import usePhase from 'api/phases/usePhase';
+
+import useFeatureFlag from 'hooks/useFeatureFlag';
+
+import T from 'components/T';
+
+import { FormattedMessage } from 'utils/cl-intl';
+
+import messages from '../../../messages';
+
+import StatusButton from './StatusButton';
+
+const StatusText = styled.div`
+  &:first-letter {
+    text-transform: uppercase;
+  }
+`;
+
+interface Props {
+  status: IIdeaStatusData;
+  active: boolean;
+  onClick: () => void;
+}
+
+const ScreeningStatusFilter = ({ status, active, onClick }: Props) => {
+  const { phaseId } = useParams() as { phaseId: string };
+  const { data: phase } = usePhase(phaseId);
+
+  const preScreeningFeatureFlag =
+    phase?.data.attributes.participation_method === 'ideation'
+      ? 'prescreening_ideation'
+      : 'prescreening';
+
+  const preScreeningFeatureAllowed = useFeatureFlag({
+    name: preScreeningFeatureFlag,
+    onlyCheckAllowed: true,
+  });
+  const phasePrescreeningEnabled =
+    phase?.data.attributes.prescreening_enabled === true;
+
+  const [{ canDrop, isOver }, drop] = useDrop({
+    accept: 'IDEA',
+    drop: () => ({
+      type: 'status',
+      id: status.id,
+    }),
+    collect: (monitor) => ({
+      isOver: monitor.isOver(),
+      canDrop: monitor.canDrop(),
+    }),
+  });
+
+  const showAutomaticStatusTooltip =
+    status.attributes.can_manually_transition_to === false;
+
+  const prescreeningButtonIsDisabled =
+    !phasePrescreeningEnabled || !preScreeningFeatureAllowed;
+
+  const prescreeningTooltipIsDisabled =
+    phasePrescreeningEnabled && preScreeningFeatureAllowed;
```

Comment:
I've removed the status code checks here since we know code is always `prescreening` here.